### ISSUE TITLE
Maintenance and Contributions to this Repository

### ISSUE THREAD

**Issue by grokpot** [emiliorizzo/vue-d3-network#100]
This is an excellent Vue project, and I'm really impressed with the work of @emiliorizzo and @Steve2955.

However, the last commit was over two years ago. There are currently 131 forks. Some of these might be for dependency management purposes, but some might have added new features.

This leaves people like me wondering - is there something better? Has someone improved this project in their own fork? How should I proceed? What about Vue3?

I suggest the following:
1. If @emiliorizzo  or @Steve2955  are aware of a better or more maintained project, please comment in this issue and make it clear at the top of the project's Readme.
2. If someone has taken this project further which would make a better "head", please note it here in the comments.
3. If someone is willing to take on maintenance of this project, please comment in this issue.

The goal is to maintain healthy project maintenance so this excellent project can be further developed (especially with Vue3)

**Comment by Steve2955** [comment]
I completely agree with you. However I'm not really a contributor to this project. [I just fixed the readme once 😅](https://github.com/emiliorizzo/vue-d3-network/commit/c8ae5e4b310009b183bba718480d69b35b8b20c6) and also haven't used `d3` with `vue` for some time now. 

I hope @emiliorizzo can provide some support to resolve this issue. All credit goes to him!
